rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -147,8 +147,22 @@ components:
     IndexRequest:
       type: object
       required:
+        - fileLocation
+        - metacardLocation
         - irmLocation
       properties:
+        fileLocation:
+          description: >
+            This URI must support HTTP GET to retrieve the file.
+          type: string
+          format: uri
```

Comment:
✏️This is not related to your changes, but we decided to use URL instead of URI from now on